src/api/index.js

```diff
@@ -1,29 +1,26 @@
-import apiClient from 'panoptes-client/lib/api-client';
+import erasClient from 'panoptes-client/lib/eras-client';
```

Comment:
Can be deleted since not using (doesn't support `project_contributions`) and using fetch instead.